qualification/conftest.py

```diff
@@ -716,68 +750,21 @@ async def receive_baseline_correlation_products(
     return receiver
 
 
-@pytest.fixture
-def receive_tied_array_channelised_voltage_manual_start(
-    pytestconfig: pytest.Config,
-    cbf: CBFRemoteControl,
-    cbf_config: dict,
-    n_antennas: int,
-    n_channels: int,
-    int_time: float,
-    band: str,
-    core_allocator: CoreAllocator,
-) -> Generator[TiedArrayChannelisedVoltageReceiver, None, None]:
-    """Create a spead2 receive stream for ingest the tied-array-channelised-voltage streams.
-
-    This fixture does not start the receiver.
-    """
-    interface_address = get_interface_address(pytestconfig.getini("interface"))
-    use_ibv = pytestconfig.getini("use_ibv")
-    interface_gbps = float(pytestconfig.getini("interface_gbps"))
-
-    stream_names = [
-        name
-        for name, config in cbf_config["outputs"].items()
-        if config["type"] == "gpucbf.tied_array_channelised_voltage"
-    ]
-
-    # Subscribe to only as many beams as can reliably be squeezed through a
-    # 100 Gb/s adapter.
-    n_bls = n_antennas * (n_antennas + 1) * 2
-    budget = interface_gbps * 1e9 - n_bls * n_channels / int_time * 64  # 64 bits per visibility
-    adc_sample_rate = BANDS[band].adc_sample_rate
-    stream_bandwidth = adc_sample_rate * 8  # 8 bits per component
-    max_streams = min(len(stream_names), int(budget // stream_bandwidth))
-    if max_streams < 4:
-        pytest.skip("Not enough network bandwidth for two dual-pol beams")
-    else:
-        logger.info("Subscribing to %d beam streams", max_streams)
-
-    stream_names = stream_names[:max_streams]
-    cores = core_allocator.allocate(len(stream_names))
-    receiver = TiedArrayChannelisedVoltageReceiver(
-        cbf=cbf, stream_names=stream_names, cores=cores, interface_address=interface_address, use_ibv=use_ibv
-    )
-    yield receiver
-    receiver.stream_group.stop()
-
-
 @pytest.fixture
 async def receive_tied_array_channelised_voltage(
-    receive_tied_array_channelised_voltage_manual_start: TiedArrayChannelisedVoltageReceiver,
-    cbf_config: dict,
+    cbf: CBFRemoteControl,
     capture_start_streams: list[str],
 ) -> TiedArrayChannelisedVoltageReceiver:
-    """Create a spead2 receive stream for ingest the tied-array-channelised-voltage streams."""
-    receiver = receive_tied_array_channelised_voltage_manual_start
-    receiver.start()
+    """Get the spead2 receive stream for ingest the tied-array-channelised-voltage streams."""
```

Comment:
I know it was just a slight tweak to this docstring, but it seems like the middle bit is a typo?
> for ingest the

Is it meant to read something like
- "for ingest *of the [...] streams", or
- "for *ingesting the [...] streams"?

I'll wait until you're back to raise a small PR accordingly.